convert2rhel/initialize.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
```

Comment:
It's interesting that git thinks that `pkgmanager.py` was renamed to `initialize.py` when in reality it would be better git history if `pkgmanager.py` was seen as renamed to `pkgmanager/__init__.py`.  Maybe not worthhile to spend a lot of time trying to fix it but maybe if you think it's a quick change to git history to correct it, it may help when someone is looking through git history.